src/modules/esl-share/actions/media-action.ts

```diff
@@ -0,0 +1,31 @@
+import {ESLShareUrlGenericAction} from './url-generic-action';
+
+import type {ESLShareButton} from '../core/esl-share-button';
+
+@ESLShareUrlGenericAction.register
+export class ESLShareMediaAction extends ESLShareUrlGenericAction {
+  public static readonly is: string = 'media';
+
+  public static FEATURES: Record<string, number> = {
+    scrollbars: 0,
+    resizable: 1,
+    menubar: 0,
+    left: 100,
+    top: 100,
+    width: 750,
+    height: 500,
+    toolbar: 0,
+    status: 0
+  };
+
+  protected get windowFeatures(): string {
+    return Object.entries((this.constructor as typeof ESLShareMediaAction).FEATURES).map((key, value) => `${key}=${value}`).join(',');
```

Comment:
Cosmetic: split the actions for readability reasons
```suggestion
    const features = (this.constructor as typeof ESLShareMediaAction).FEATURES;
    return Object.entries(features).map((key, value) => `${key}=${value}`).join(',');
```